test_unified_ingestion.py

```
@@ -0,0 +1,314 @@
+async def test_github_repository():
+    """Test GitHub repository ingestion."""
+    print("\n🔷 TESTING GITHUB REPOSITORY INGESTION")
+    print("=" * 60)
+    
+    # Use a small test repository
+    repo_url = "https://github.com/octocat/Hello-World"
+    
+    try:
+        result = await ingest_github_repo(repo_url)
+        
+        print(f"✅ GitHub ingestion successful!")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)